tests/boot/boot_from_pxe.pm

```diff
@@ -1,6 +1,6 @@
 # SUSE's openQA tests
 #
-# Copyright © 2012-2018 SUSE LLC
+# Copyright ©  2012-2018 SUSE LLC
```

Comment:
please revert

Reply:
I think it already went back to origin state